charts/emissary-ingress/templates/deployment.yaml

```diff
@@ -173,7 +173,7 @@ spec:
               containerPort: {{ .Values.adminService.port }}
           env:
             - name: AMBASSADOR_GRPC_METRICS_SINK
-              value: {{ include "ambassador.fullname" . }}:8006
+              value: {{ include "ambassador.fullname" . }}-agent:http
```

Comment:
@LukeShu we cannot use a non-integer here for the port in this environment variable because it will break the Envoy config generation here: https://github.com/emissary-ingress/emissary/blob/128bce4fe68dc0f7912b7e28acaedae50b449f4c/python/ambassador/envoy/v3/v3bootstrap.py#L205

```
>>> urlparse("//foobar:http").port
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/Cellar/python@3.9/3.9.10/Frameworks/Python.framework/Versions/3.9/lib/python3.9/urllib/parse.py", line 178, in port
    raise ValueError(message) from None
ValueError: Port could not be cast to integer value as 'http'
```